x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/query/index.test.tsx

```diff
@@ -109,11 +109,12 @@ describe('Timeline', () => {
       }
     );
 
+    // @ts-ignore
```

Comment:
Is this still needed?

Reply:
@janmonschke so while this could be removed, I'm a bit confused of why this wasn't an issue before this PR. The only thing I did was removing 3 props (`onEventClosed`, `showExpandedDetails` and ``) and now this error pops up:
```
Property dispatch is missing in type but required in type DispatchProp<AnyAction>
```

I believe this is coming from the way we type the Props for these components: we do a union with this PropsFromRedux thingy
```
type PropsFromRedux = ConnectedProps<typeof connector>;
```
and dispatch must be in there, even though we don't need it. I can add it like this for example
```
props: {
    dispatch: {} as Dispatch,
    {...}
};
```

but then I'd have to update the snapshot, and while it might be more correct than what we have now, it feels weird, doesn't it?

WDYT?

Reply:
All good, I thought this was part of prod code and not just in a test.

Reply:
just fyi, I decided to go ahead with removing the `ts-ignore` and updating the snapshots (see this [last commit](https://github.com/elastic/kibana/pull/187846/commits/768d6d8cbc253049be0d4df2c1dc5b7f6096c953))